zubhub_backend/zubhub/activitylog/serializers.py

```diff
@@ -0,0 +1,35 @@
+from creators.models import Creator
+from rest_framework import serializers
+
+from .models import Activitylog
+from creators.serializers import CreatorMinimalSerializer
+
+class CreatorInfoSerializer(serializers.ModelSerializer):
+    tags = serializers.SlugRelatedField(slug_field="name",
+                                        read_only=True,
+                                        many=True)
+    # badges = serializers.SlugRelatedField(slug_field="badge_title",
+    #                                     read_only=True,
+    #                                     many=True)
+
+    class Meta:
+        model = Creator
+        fields = ('id', 'username', 'followers_count', 
+                  'following_count', 'projects_count', "tags", 
+                #   "badges"
```

Comment:
Consider removing this field as well, if you are not planning on using it..

Reply:
Done.